src/tests/function/floor.rs

```diff
@@ -0,0 +1,85 @@
+use crate::*;
+
+test_case!(floor, async move {
+    use Value::F64;
+
+    let test_cases = vec![
+        (
+            "CREATE TABLE SingleItem (id INTEGER PRIMARY KEY)",
+            Ok(Payload::Create),
+        ),
+        (
+            r#"INSERT INTO SingleItem VALUES (0)"#,
+            Ok(Payload::Insert(1)),
+        ),
+        (
+            "SELECT FLOOR(0.3) AS floor FROM SingleItem",
+            Ok(select!(
+                "floor";
+                F64;
+                0.3_f64.floor()
+            )),
+        ),
+        (
+            "SELECT FLOOR(-0.8) AS floor FROM SingleItem",
+            Ok(select!(
+                "floor";
+                F64;
+                (-0.8_f64).floor()
+            )),
+        ),
+        (
+            "SELECT FLOOR(10) AS floor FROM SingleItem",
+            Ok(select!(
+                "floor";
+                F64;
+                f64::from(10).floor()
+            )),
+        ),
```

Comment:
These test cases are identical from the view of our project.
It would be good to consider merging into a one. `SELECT FLOOR(0.3), FLOOR(-0.8), FLOOR(10) FROM SingleItem`

ref https://github.com/gluesql/gluesql/pull/289#discussion_r690050617